packages/app/components/pack_table/Table.tsx

```diff
@@ -72,6 +72,21 @@ export const TableContainer = ({
   if (error) return <ErrorMessage message={error} />;
 
   const isWeb = Platform.OS === 'web';
+
+  const headerRow = isWeb ? [
```

Comment:
We should eliminate these columns and just use the zeego dropdown you made on web, its cleaner and makes UI easier to manage